apps/snitch_core/test/data/schema/shipping_rules/shipping_rule_identifier_test.exs

```diff
@@ -0,0 +1,58 @@
+defmodule Snitch.Data.Schema.ShippingRuleIdentifierTest do
+  use ExUnit.Case, async: true
+  use Snitch.DataCase
+
+  import Snitch.Factory
+
+  alias Snitch.Data.Schema.ShippingRuleIdentifier, as: SRIdentifier
+
+  setup do
+    si = insert(:shipping_identifier)
+    params = Map.from_struct(si)
+
+    [params: params]
+  end
+
+  describe "changeset/2" do
+    test "successful for valid params", %{params: params} do
```

Comment:
keep the names for changeset function related statements as returns valid or invalid always.